$NetBSD: patch-al,v 1.1 2001/03/23 16:20:56 agc Exp $

Patch from Wolfgang Solfrank:

A minor correction to the computation of the SIO rate.

--- console.c.orig	Sun Feb 11 03:52:30 2001
+++ console.c	Fri Mar 23 16:01:53 2001
@@ -1257,7 +1257,8 @@
         } /* end for(dev) */
 
         /* Wait for a file descriptor to become ready */
-#ifdef WIN32
+#ifndef	__notdef__
+/*#ifdef WIN32*/
 	{
 	    struct timeval tv={0,500000};	/* half a second */
 	    rc = select ( maxfd+1, &readset, NULL, NULL, &tv );
